### PR TITLE
Update browser-support.md

### PR DESCRIPTION
**Pull request by dipinn26592**
We need to call out the browser does not support both video and screen sharing together. only one at a time.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/includes/browser-support.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ Teams fully supports the following Internet browsers, with noted exceptions for
 
 <sup>2</sup> Blur my background isn't available when you run Teams in a browser. This feature is only available in the Teams desktop client.
 
-<sup>3</sup> Teams meetings on browsers are limited to a single incoming video feed of active speaker.
+<sup>3</sup> Teams meetings on browsers are limited to a single stream. Either incoming video feed of active speaker or Screen Sharing.
```
Comment (JohanFreelancer9) [inline Teams/includes/browser-support.md:19]:
```suggestion
<sup>3</sup> Teams meetings on browsers are limited to a single stream; either incoming video feed of the current speaker or screen sharing.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @dipinn26592, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by dipinn26592** [comment]
committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@chuckedmonson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by chuckedmonson** [comment]
@SerdarSoysal -- reassigning to your team. Thanks.

**Comment by tiburd** [comment]
reviewed + confirmed

**Pull request merged**